backend/test/neo4j.test.ts

```diff
@@ -0,0 +1,175 @@
+import { gql } from 'apollo-server';
+import { createTestClient } from 'apollo-server-testing';
+
+import { makeNeo4jTestServer } from '../src/apollo';
+import { CreateTodoArguments, TodosArguments } from '../src/Interfaces';
+
+let query: any;
+let mutate: any;
+/**
+ * Setup test data
+ */
+describe('queries and mutations', () => {
+    beforeEach(async () => {
+        const neo4jTestServer = makeNeo4jTestServer({});
+        const client = createTestClient(neo4jTestServer);
+        query = client.query;
+        mutate = client.mutate;
+    });
+
+    describe('Retrieve User "Anderson"', () => {
```

Comment:
How about:
```suggestion
    describe('getUserByName', () => {
```
like the name of the query?